rules/windows/discovery_security_software_wmic.toml

```diff
@@ -52,32 +52,39 @@ If enabling an EQL rule on a non-elastic-agent index (such as beats) for version
 risk_score = 47
 rule_id = "6ea55c81-e2ba-42f2-a134-bccf857ba922"
 severity = "medium"
-tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Discovery", "Resources: Investigation Guide", "Data Source: Elastic Endgame", "Data Source: Elastic Defend"]
+tags = ["Domain: Endpoint",
+        "OS: Windows",
+        "Use Case: Threat Detection",
+        "Tactic: Discovery",
+        "Resources: Investigation Guide",
+        "Data Source: Elastic Endgame",
+        "Data Source: Elastic Defend"
+        ]
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
-   (process.name:"wmic.exe" or process.pe.original_file_name:"wmic.exe") and
-    process.args:"/namespace:\\\\root\\SecurityCenter2" and process.args:"Get"
+(process.name : "wmic.exe" or process.pe.original_file_name : "wmic.exe") and
+process.args : "/namespace:\\\\root\\SecurityCenter2" and process.args : "Get" and
+not process.parent.executable : "?:*\\JetBrains*" and not process.parent.name : ("jp2launcher.exe", "Remote Access.exe",
```

Comment:
interestingly, almost all of the hits from telemetry are from IDE's or some other code support. The problem with excluding those however, is that its a simple convenient way to proxy execution through an IDE. It would be suspicious if users were _not_ supposed to be doing dev on their systems.

So the question is - is excluding them better off as an user defined exclusion or a global rule modification such as this?

Reply:
could also make it a new terms with a really short window, like 24h?

Reply:
Reverted changes, and added it to the umbrella new_terms rule!